build/crds/devices/devices_v1beta1_devicestatus.yaml

```diff
@@ -0,0 +1,179 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.16.5
```

Comment:
As #6549 has been merged, the controller-gen version is not v0.17.3. Please rebase the newest master branch and generate crds again.